src/apollo/config.js

```diff
@@ -3,7 +3,7 @@ import { ApolloClient, InMemoryCache, createHttpLink } from '@apollo/client';
 
 const link = createHttpLink({
   /** our graphql endpoint */
-  uri: 'http://localhost:4142/graphql',
+  uri: 'https://apollo-server-simple-demo.herokuapp.com/graphql',
```

Comment:
would be worth adding a note / comment to remind that the actual graphql endpoint is `http://localhost:4142/graphql`